Client/admin_page.html

```diff
@@ -90,7 +90,14 @@
         <th>ID</th>
         <th>Name</th>
         <th>Image</th>
+        <th></th>
       </tr>
+    <tr>
+        <td><input type="number" name="id"/></td>
+        <td><input type="text" name="name"/></td>
+        <td><input type="url" name="img-url"/></td>
```

Comment:
Delete _url_ input.
There is no ability to upload image by setting image's url.
Image will be uploaded in the Category edit form. This will be addressed in #24